packages/aws-client/src/DynamoDbSearchQueue.ts

```diff
@@ -1,17 +1,20 @@
 import * as AWS from 'aws-sdk';
 import { dynamodbDocClient } from './services';
 
+type searchType = 'scan' | 'query';
+type searchParams = AWS.DynamoDB.DocumentClient.ScanInput | AWS.DynamoDB.DocumentClient.QueryInput;
+
 /**
  * Class to efficiently search all of the items in a DynamoDB table, without loading them all into
  * memory at once.  Handles paging.
  */
 class DynamoDbSearchQueue {
   private readonly dynamodbDocClient: AWS.DynamoDB.DocumentClient;
-  private readonly searchType: 'scan';
-  private readonly params: AWS.DynamoDB.DocumentClient.ScanInput;
+  private readonly searchType: searchType;
+  private readonly params: searchParams;
   private items: Array<AWS.DynamoDB.DocumentClient.AttributeMap|null>;
 
-  constructor(params: AWS.DynamoDB.DocumentClient.ScanInput, searchType: 'scan' = 'scan') {
+  constructor(params: AWS.DynamoDB.DocumentClient.ScanInput, searchType: searchType = 'scan') {
```

Comment:
Probably need more unit tests for the `query` case?

Reply:
Please advise whether 8e6b65ded63953aa3ad426a1e956efce18cdc6e3 is sufficient

Reply:
The test coverage report for this file specifically was:

` DynamoDbSearchQueue.ts   |      76 |    70.59 |      80 |   71.43 | 32-39`

So the only uncovered lines are 32-39, which are nothing related to what I changed